### DISCUSSION TITLE
Video Tutorial

### DISCUSSION THREAD

**Discussion by nicolatrentini** [iisgiua/giuaschool#492] [Richieste di aiuto] [answered]
buongiorno vorrei richiedere un video tutorial su come installare il registro Giua@School
spero che mi risponderete con un video e con una risposta .

Cordiali Saluti ,
Nicola Trentini

**Comment by giadaschiavone** [comment]
Ciao, mi fa piacere risentirti. Scrivimi in privato, dobbiamo urgentemente parlare.

Grazie in anticipo,
prof.ssa Giada Schiavone

**Reply by nicolatrentini** [reply]
ok ti ho scritto

**Comment by trinko** [comment, marked as answer]
> buongiorno vorrei richiedere un video tutorial su come installare il registro Giua@School spero che mi risponderete con un video e con una risposta .
> 
> Cordiali Saluti , Nicola Trentini

Ci dovrebbe essere qualcuno che offre supporto per l'installazione.
Prova a scrivere un'email al seguente indirizzo, che è stato più volte citato su questo forum:

giuaschoolregistro@gmail.com

**Reply by nicolatrentini** [reply]
ciao Antonello ,
il supporto non mi risponde . 
lo potrei contattare tramite Google Chat?

**Reply by trinko** [reply]
Non è un servizio di supporto e non è gestito dalla mia scuola. 
Immagino effettuino questi interventi quando è possibile. Di più non so.

**Comment by nicolatrentini** [comment]
Ok
Ma non mi serve più perché ho capito come si installa il Registro

Il gio 27 apr 2023, 00:15 Antonello Dessì ***@***.***> ha
scritto:
